### ISSUE TITLE
why loss_dict is None？

### ISSUE THREAD

**Issue by 281LinChenjian** [hpcaitech/Open-Sora#630]
当我训练Open-Sora v1.2时，我发现在在此处加入print语句时loss_dict和loss均为Nan，请问这是因为使用了ColossalAI分布式框架后出现的正常情况吗？就如同之前有人issue里提问说他发现所有的model.params 的grad也都为None

                # == diffusion loss computation ==
                with timers["diffusion"] as loss_t:
                    loss_dict = scheduler.training_losses(model, x, model_args, mask=mask)
                if record_time:
                    timer_list.append(loss_t)

                # == backward & update ==
                with timers["backward"] as backward_t:
                    print("loss_dict:",loss_dict)
                    loss = loss_dict["loss"].mean()
                    print("loss:",loss)
                    booster.backward(loss=loss, optimizer=optimizer)

 我在scheduler.training_losses里面添加print语句时发现也不会输出任何东西，是函数没有正确被调用吗，还是说也是属于正常现象？
    def training_losses(self, model, x_start, model_kwargs=None, noise=None, mask=None, weights=None, t=None):
        print(123456)

**Comment by JThh** [comment]
Please do not use `print`; use `logging.info()` for printing help messages.
